ci/prow/config.yaml

```diff
@@ -314,6 +314,32 @@ presubmits:
       - name: github-token
         secret:
           secretName: covbot-token
+  - name: pull-knative-serving-perf-tests
+    agent: kubernetes
+    context: pull-knative-serving-perf-tests
+    always_run: false
+    rerun_command: "/test pull-knative-serving-perf-tests"
+    trigger: "(?m)^/test (all|pull-knative-serving-perf-tests),?(\\s+|$)"
+    labels:
+      preset-test-account: "true"
+    skip_branches:  # Skip these branches, as test isn't available.
```

Comment:
You probably don't need this if they're not running on presubmit. If they do, then you'll need to skip branch release-0.3 until this is merged into the branch (or skip forever, if it never gets merged into the branch).

Reply:
Added 0.3. Will keep this in from now, instead of remebering to add it later :D